StaggeredCoupledProcesses/TH/ThermalDensityDrivenFlow2D/thermal_gravity_driven_flow2d.prj

```diff
@@ -0,0 +1,304 @@
+            <name>rho</name>
+            <type>Constant</type>
+            <value>1.6e+3</value>
+        </parameter>
+        <parameter>
+            <name>T0</name>
+            <type>Constant</type>
+            <value>293</value>
+        </parameter>
+        <parameter>
+            <name>TB_left</name>
+            <type>Constant</type>
+            <value>373.0</value>
+        </parameter>
+        <parameter>
+            <name>TB_right</name>
+            <type>Constant</type>
+            <value>293.0</value>
+        </parameter>
+        <parameter>
+            <name>solid_thermal_expansion</name>
+            <type>Constant</type>
+            <value>1.e-5</value>
+        </parameter>
+        <parameter>
+            <name>biot_const</name>
+            <type>Constant</type>
+            <value>1.0</value>
+        </parameter>
+    </parameters>
+
+    <process_variables>
+        <process_variable>
+            <name>temperature</name>
+            <components>1</components>
+            <order>1</order>
+
+            <initial_condition>T0</initial_condition>
+
+            <boundary_conditions>
+                <boundary_condition>
+                    <geometrical_set>gravity_dr2D</geometrical_set>
+                    <geometry>right</geometry>
+                    <type>Dirichlet</type>
+                    <parameter>TB_right</parameter>
+                </boundary_condition>
+                <boundary_condition>
+                    <geometrical_set>gravity_dr2D</geometrical_set>
+                    <geometry>left</geometry>
+                    <type>Dirichlet</type>
+                    <parameter>TB_left</parameter>
+                </boundary_condition>
+            </boundary_conditions>
+        </process_variable>
+
+        <process_variable>
+            <name>pressure</name>
+            <components>1</components>
+            <order>1</order>
+
+            <initial_condition>p0</initial_condition>
+
+            <boundary_conditions>
+                <boundary_condition>
+                    <geometrical_set>gravity_dr2D</geometrical_set>
+                    <geometry>top</geometry>
+                    <type>Dirichlet</type>
+                    <parameter>p_bc</parameter>
+                </boundary_condition>
+            </boundary_conditions>
+
+        </process_variable>
+    </process_variables>
+
+    <nonlinear_solvers>
+        <nonlinear_solver>
+            <name>basic_picard_T</name>
+            <type>Picard</type>
+            <max_iter>10</max_iter>
+            <linear_solver>linear_solver_T</linear_solver>
+        </nonlinear_solver>
+        <nonlinear_solver>
+            <name>basic_picard_H</name>
+            <type>Picard</type>
+            <max_iter>10</max_iter>
+            <linear_solver>linear_solver_H</linear_solver>
+        </nonlinear_solver>
+    </nonlinear_solvers>
+
+    <linear_solvers>
+        <linear_solver>
+            <name>linear_solver_T</name>
```

Comment:
By the order of definitions of processes. The first one for the first process, the second for the second process, etc.